docs/configuration.rst

```diff
@@ -84,6 +85,49 @@ Note that, equivalently, you can add these during webapp configuration:
             config.include('pyramid_swagger')
 
 
+user_formats (Swagger 2.0 only)
+---------------------------------------
+
+The option ``user_formats`` provides user defined formats which can be used
+for validations/format-conversions. This options can only be used via webapp
+configuration.
+
+Sample usage:
+
+.. code-block:: python
+
+        def main(global_config, **settings):
+            # ...
+            settings['pyramid_swagger.user_formats'] = [user_format]
+
+
+And, ``user_format`` used above can be defined like this:
+
+.. code-block:: python
+
+        import base64
+        from pyramid_swagger.tween import SwaggerFormat
+        user_format = SwaggerFormat(format='base64',
```

Comment:
The only link i can think of is a bravado-core github link for that [snippet](https://github.com/Yelp/bravado-core/blob/master/bravado_core/formatter.py#L72). User can anytime fetch the docstring of SwaggerFormat though.